### PR TITLE
Dispute policy for ID prefix reservations

### PR DESCRIPTION
**Pull request by diverdan92**

### CODE REVIEW

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/ID-Prefix-Reservation.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/diverdan92-patch-1/docs/reference/ID-Prefix-Reservation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/id-prefix-reservation?branch=pr-en-us-932) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C12%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201806122157229849-932%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3xZhelWpezZfrjmDoyEjvA3RljN0Ut6MYUTvO5nEV7s%3D&st=2018-06-12T21%3A53%3A24Z&se=2018-07-13T21%3A58%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/ID-Prefix-Reservation.md [hunk 1]
```diff
@@ -87,3 +87,7 @@ When reviewing any application for ID prefix reservation, the [nuget.org](https:
 If a third party feed provider is interested in implementing their own service to provide prefix reservations, you can do so by modifying the search service in the NuGet V3 feed providers. The addition in the feed search service is to add the *verified* property, with examples for the V3 feeds below. The NuGet client will not support the added property in the V2 feed.
 
 For more information, see the [documentation about the API's search service](../api/search-query-service-resource.md).
+
+## Package ID Prefix Reservation Dispute Policy
+If you believe an owner on [NuGet.org](https://nuget.org) was assigned a package ID prefix reservation that goes against the above listed criteria, or infringes on any trademarks or copyrights, please email [support@nuget.org](mailto:support@nuget.org) with the ID prefix in question, the owner of the ID prefix, and the reason for disputing the assigned prefix reservation.
```
Comment (Phrohdoh) [inline docs/reference/ID-Prefix-Reservation.md]: Should probably use `https://www.nuget.org` for consistency and so everything links to the www subdomain.

Comment (diverdan92) [reply docs/reference/ID-Prefix-Reservation.md]: Should be fixed. Thanks!

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/ID-Prefix-Reservation.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/diverdan92-patch-1/docs/reference/ID-Prefix-Reservation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/id-prefix-reservation?branch=pr-en-us-932) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C9%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201807091730563615-932%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dw3uNOxLR3b%2BvzwYB0%2FS%2FPLM8Tp3PGM9%2Bl8zy%2FhRGcI%3D&st=2018-07-09T17%3A27%3A00Z&se=2018-08-09T17%3A32%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**